### PR TITLE
Fix code coverage reporting

### PR DESCRIPTION
**Pull request by yjpa7145**

### CODE REVIEW

File: example/package.json [hunk 1]
```diff
@@ -15,14 +15,15 @@
     "timeout": "15m"
   },
   "scripts": {
-    "test": "../node_modules/.bin/ava && for x in lambdas/*; do cd $x && npm run test && cd -; done",
-    "int-test": "../node_modules/.bin/jasmine && npm run parallel-tests",
-    "parallel-tests": "sh scripts/tests-parallel.sh",
-    "redeploy-test": "../node_modules/.bin/jasmine spec/standalone/redeployment/*.js",
     "all-tests": "npm run parallel-tests && ../node_modules/.bin/jasmine && npm run redeploy-test",
+    "install-python-deps": "for x in lambdas/*; do cd $x && npm run install-python-deps || true && cd -; done",
```
Comment (Jkovarik) [inline example/package.json:19]: Is this just an alpha reorder?

Comment (yjpa7145) [reply example/package.json:19]: Yeah, sorry, the interesting change is to the `test` and `test:ava` scripts.

File: tasks/add-missing-file-checksums/package.json [hunk 2]
```diff
@@ -19,9 +19,11 @@
     "test": "tests"
   },
   "scripts": {
+    "clean": "rm -rf dist",
```
Comment (Jkovarik) [inline tasks/add-missing-file-checksums/package.json:22]: 👍

File: packages/api-client/.npmignore [hunk 3]
```diff
@@ -1 +1,2 @@
+/nyc.config.js
```
[the remark this answers is not in the document]
Comment (Jkovarik) [reply packages/api-client/.npmignore:1]: And really for the nyc config files as well, perhaps more so given we probably won't want much customization.

[the remark this answers is not in the document]
Comment (yjpa7145) [reply packages/api-client/.npmignore:1]: The problem is that they are _subtly_ different. Some ignore `test`, some ignore `tests`, some ignore `tsconfig`, etc. The same goes for the `nyc.config.js` files. Some of our packages are just JS, some are TS, and the TS ones may put files in `dist` or they may end up at the root.

[the remark this answers is not in the document]
Comment (Jkovarik) [reply packages/api-client/.npmignore:1]: Right, but things like the test/tests difference are variances that probably aren't meaningful and should be standardized, and ignoring tsconfig (and the others) in a js package that is moving to ts *probably* isn't an issue for a .npm ignore.       

Obviously nyc config would wait until all the things are TS.   

Again, not directly advocating, just hate to see that kind of repetition if we can avoid it sanely.

**Pull request merged**